forge/db/migrations/20230808-01-add-access-token-name.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
This migration will need to be renamed. We have already merged `20230814-01-add-description-to-applications` so this file will need a later date.